docs/index.html

```diff
@@ -64,10 +64,10 @@ <h2 class="libraries-section__title">Fundamental Library</h2>
                         class="libraries-section__logo" />
 
                     <h3 class="libraries-section__item-title">Fundamental Library Styles</h3>
-                    <p class="libraries-section__text">Delivers a Fiori 3.0 component library for building SAP user interfaces with any web technology, including artifacts like a UI kit, SAP icons, SAP font and component specifications.</p>
+                    <p class="libraries-section__text">Delivers a Fiori 3.0 component library for building SAP user interfaces with any web technology.</p>
                 </div>
                 <div class="libraries-section__links">
-                    <a class="libraries-section__more" href="{{site.baseurl}}/getting-started.html" target="_blank">Learn More <span class="vh">about Fundamental Library Styles</span></a>
+                    <a class="libraries-section__more" href="{{site.baseurl}}/getting-started.html">Learn More <span class="vh">about Fundamental Library Styles</span></a>
```

Comment:
Removed target blank here because it's the same domain - shouldn't open in a new tab.

Reply:
@jbadan this was a requirement actually

Reply:
Why?